dnf5/commands/install/install.hpp

```diff
@@ -40,6 +42,15 @@ class InstallCommand : public Command {
     std::vector<std::string> pkg_specs;
     std::vector<std::string> pkg_file_paths;
     std::vector<libdnf::rpm::Package> cmdline_packages;
+
+    std::unique_ptr<AdvisoryOption> advisory_name{nullptr};
```

Comment:
I would like to ask you whether we can skip such a declarations?

Reply:
I think we need them to be a part of the `InstallCommand` class.

Or do you mean to skip the `nullptr` constructor argument? Yes can update it to:
`std::unique_ptr<AdvisoryOption> advisory_name;`

Reply:
I have removed the `nullptr`'s and also deduplicated a bit of code for `UPGRADE_ALL` and `UPGRADE_ALL_MINIMAL`.
Please take a look.